### ISSUE TITLE
Installation/Upgrade process is confusing

### ISSUE THREAD

**Issue by aybe** [git-for-windows/git#4020]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
git version 2.37.3.windows.1
cpu: x86_64
built from commit: c4992d4fecabd7d111726ecb37e33a3ccb51d6f1
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
Microsoft Windows [Version 10.0.19044.1889]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
Editor Option: VisualStudioCode
Custom Editor Path:
Default Branch Option:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Enabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Enabled
```

### Details

I decided to reinstall the program because I wanted to have the context menu entries for Explorer:

<img width="78" alt="UYfOOl4Tty" src="https://user-images.githubusercontent.com/1537591/190449299-4c2b56c1-2b76-4290-a928-510112d9e125.png">

But it turns out that installing over an existing installation, i.e. upgrade, the setup simply does not show the numerous options to customize the installation.

Fine, so I've uninstalled it then installed the new version, the wizard for configuring the software is now visible, however, it still complained about an existing installation.

In short,

- upgrading an installation does not show the wizard for configuring options
- uninstalling leaves leftovers
- installing again complains about leftovers

Maybe that could be improved somehow?

**Comment by dscho** [comment]
> But it turns out that installing over an existing installation, i.e. upgrade, the setup simply does not show the numerous options to customize the installation.

Yes, that's because this checkbox is checked: 
![image](https://user-images.githubusercontent.com/127790/190543414-5fa15d8f-8286-4ec4-ad31-29af7dcb9b7c.png)

**Comment by aybe** [comment]
Got it, thank you!
